src/scripts/editorsidebar.js

```diff
@@ -149,7 +149,9 @@ define(["datetime", "jQuery", "material-icons"], function(datetime, $) {
         nodesToLoad = [], selectedNodeId = null, $.jstree.destroy(), $(".libraryTree", page).jstree({
             plugins: ["wholerow"],
             core: {
-                check_callback: !0,
+                // Disable animations because jQuery slim does not support them
```

Comment:
Can we migrate to normal jQuery with no issues to support animations?

Reply:
I'm not opposed to that, but I don't think it should be included in the scope of the webpack changes. This is only dealing with animating the metadata tree view which seems to have been disabled by Emby by editing the jstree source.